ts/views/conversation_view.ts

```diff
@@ -547,6 +547,9 @@ export class ConversationView extends window.Backbone.View<ConversationModel> {
     // setupCompositionArea
     window.reduxActions.composer.resetComposer();
 
+    window.reduxStore.getState().composer.shouldSendHighQualityAttachments =
```

Comment:
We can't just overwrite state in redux like this.

The global setting state should already live in the `items` duck so how about we create a selector that derives the state?

Reply:
whew, such a relief; this was far too much a hack for my taste, but I was poking around for the fuse in the dark; thanks a bunch for the `ducks` reference, would have taken me way too long to find it.